scripts/fastsp_script/fsp_test.py

```diff
@@ -2,39 +2,98 @@
 import sys
 
 """
-###remember sys.argv - can create variables using indexes with sys.argv!  
-    
+###remember sys.argv - can create variables using indexes with sys.argv!      
 """
 
 dir = "perturbed_alignments_Drosophila_AA/EMGT00050000000018.Drosophila.001_AA"
 slash = "/"
+fas = ".fasta"
 
-#print(sys.argv)
-
-outfile = output_fs.txt
-#will need to be .csv later 
+infile = open("output_fs.txt", "w+")
+infile_name = "output_fs.txt"
+outfile = open("parsed_fs_output.csv","w")
+header = "reference,estimation,datatype,sp_score,tc"+ "\n"
+com = ","
+outfile.write(header)
+space = " "
 
 aa_ls_dir = os.listdir(dir)
-
-for file in aa_ls_dir:
-    if file.endswith(".fasta"):
-        #print (file)
-        for file2 in aa_ls_dir:
-            if file2.endswith(".fasta"):
-                if file != file2:
-                    print(file,file2)
-                    #print(dir+slash+file + dir+slash+file2)
-                    
-                    fasta1 = dir+slash+file
-                    fasta2 = dir+slash+file2
-                    
+                                
+for i in range(50):
+    for j in range (i,50):
+        if i != j:
+        
+            file = aa_ls_dir[i]
```

Comment:
There is a concept here that we will discuss over zoom - it is better for your *computer's memory* to not keep defining this. When you have nested loops, you want to define variables associated with each looping variable (i,j) as infrequently as possible. So, this line 26 should be BETWEEN lines 22/23. This script does not rise to the level of complexity that would cause problems, but it's a good habit to get into. For an explanation of things like this going really really badly: https://en.wikipedia.org/wiki/Thrashing_(computer_science)